site/content/docs/4.3/forms/validation.md

```diff
@@ -98,31 +94,10 @@ Custom feedback styles apply custom colors, borders, focus styles, and backgroun
       </div>
     </div>
   </div>
-  <button class="btn btn-primary" type="submit">Submit form</button>
+  <div class="col-12">
+    <button class="btn btn-primary" type="submit">Submit form</button>
+  </div>
 </form>
-
-<script>
-// Example starter JavaScript for disabling form submissions if there are invalid fields
-(function () {
-  'use strict';
-
-  // Fetch all the forms we want to apply custom Bootstrap validation styles to
-  var forms = document.querySelectorAll('.needs-validation');
-
-  // Loop over them and prevent submission
-  Array.prototype.slice.call(forms)
-    .forEach(function (form) {
-      form.addEventListener('submit', function (event) {
-        if (!form.checkValidity()) {
-          event.preventDefault();
-          event.stopPropagation();
-        }
-
-        form.classList.add('was-validated');
-      }, false);
-    });
-})();
-</script>
```

Comment:
I don't think this script should be hidden. If you want it, please create another PR.

Reply:
I moved this to the end of the file to fix form validation for https://twbs-bootstrap.netlify.com/docs/4.3/forms/validation/?#custom-styles. Shall make another PR for this

Reply:
I missed [the comment](https://github.com/twbs/bootstrap/pull/28517#discussion_r364956887). `ul.row` raises the CSS priority, so for example overwriting its `padding-left` may cause confusion.

```html
<ul class="row custom-class">...</ul>
```
```css
.custom-class {
   padding: 1rem;
}
```

It is a safe way for users to add `.list-unstyled` deliberately. Or, I think as in the first @MartijnCuppens 's suggestion, applying rest list style to `.row` without limiting to `ul` and `li` may not cause confusion of CSS priority. e.g.:

```css
.row {
  ...
  padding: 0;
  list-style: none;
}
```

Reply:
Reverted this